x-pack/plugins/observability_solution/observability_ai_assistant_management/public/hooks/use_get_knowledge_base_entries.ts

```diff
@@ -20,13 +20,13 @@ export function useGetKnowledgeBaseEntries({
 }) {
   const { observabilityAIAssistant } = useKibana().services;
 
-  const observabilityAIAssistantApi = observabilityAIAssistant?.service.callApi;
+  const observabilityAIAssistantApi = observabilityAIAssistant.service.callApi;
 
   const { isLoading, isError, isSuccess, isRefetching, data, refetch } = useQuery({
     queryKey: [REACT_QUERY_KEYS.GET_KB_ENTRIES, query, sortBy, sortDirection],
     queryFn: async ({ signal }) => {
-      if (!observabilityAIAssistantApi || !signal) {
-        return Promise.reject('Error with observabilityAIAssistantApi: API not found.');
+      if (!signal) {
+        return Promise.reject('Abort signal missing');
```

Comment:
understood it's not a change, but can you reject with `new Error` here?

Reply:
Absolutely! I also wondered why we use `reject` here and was going to change but also didn't want to change too many unrelated things.
Will fix